wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/Robot.java

```diff
@@ -17,8 +17,6 @@
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
 
-  private RobotContainer m_robotContainer;
```

Comment:
Any specific reason why this was was removed? It doesn't match up with the other command-based examples/templates

Reply:
It was moved locally due to build failures. But with the addition of the chooser, I brought it back.